### ISSUE TITLE
Update system python version on Cheyenne

### ISSUE THREAD

**Issue by danielabdi-noaa** [ufs-community/ufs-srweather-app#402]
## Description

 
The system python3 version on Cheyenne is `3.4.10` and is below what is required by SRW app which is >3.6.
The requirement on the system python is a new thing that came from transition of the workflow from bash to python.
With python `3.4.10` crontab deletion failures happen, literally due to a syntax error. You see a message like below that
indicates it does not understand f-strings of python that were introduced in 3.5 i think.
```
  File "/glade/scratch/epicufsrt/jenkins/workspace/fs-srweather-app_pipeline_PR-378__2/ush/get_crontab_contents.py", line 63
    (_, __crontab_contents__, _) = run_command(f"""{__crontab_cmd__} -l""")
                                                                         ^
SyntaxError: invalid syntax
```
  

## Solution

Update the system python version to 3.6 atleast. Best would be to match what is in conda i think.

## Requirements**

System python3 version >3.6

## Acceptance Criteria (Definition of Done)

Update the system python3 version to atleast 3.6

## Dependencies (optional)

None

## Alternative Solutions (optional)

There is currently a workaround in place to activate conda specifically for Cheyenne even when there is no task to run, i.e. for deleting crontab content at the end of a workflow run. This could be extended to all other systems, in which case there will be no requirement on the system python version.

**Comment by zmoon** [comment]
f-strings were new in 3.6: https://docs.python.org/3/whatsnew/3.6.html#pep-498-formatted-string-literals

**Comment by natalie-perlin** [comment]
For the SRW purposes, a miniconda3/4.12.0 with python 3.9.12 was installed on Cheyenne in a common space, maintained by EPIC. This miniconda3 installations also has regional_workflow environment with the packages for running the SRW and plotting (using sample plotting routines).

Miniconda3 installation location:
/glade/work/epicufsrt/contrib/miniconda3/4.12.0

Loading the module:
```
module use /glade/work/epicufsrt/contrib/miniconda3/modulefiles
module load miniconda3/4.12.0
```

Starting the regional_workflow environment after loading the miniconda3 module:
`conda activate regional_workflow`

**Comment by MichaelLueken** [comment]
Closed with #404
